generic-sensor/generic-sensor-tests.js

```diff
@@ -10,7 +10,8 @@ const properties = {
   'AbsoluteOrientationSensor' : ['timestamp', 'quaternion'],
   'RelativeOrientationSensor' : ['timestamp', 'quaternion'],
   'GeolocationSensor' : ['timestamp', 'latitude', 'longitude', 'altitude',
-                         'accuracy', 'altitudeAccuracy', 'heading', 'speed']
+                         'accuracy', 'altitudeAccuracy', 'heading', 'speed'],
+  'ProximitySensor' : ['timestamp', 'max']
```

Comment:
Are you missing the `distance` and `near` attribute?

Reply:
If the physical object is outside the sensing range, the attribute `distance` and `near` must return null, it conflicts the common function `assert_reading_not_null`, so I will add tests for them separately.